### ISSUE TITLE
Support switching on/off at certain times

### ISSUE THREAD

**Issue by jamesefhawkins** [papercups-io/papercups#243]
We usually have team members awake 24 hours a day able to respond to user messages in papercups.

However, at the moment, we don't!

We'd love the ability to edit the hours when papercups is on/off for situations when support is / is not available.

**Comment by reichert621** [comment]
good idea @jamesefhawkins! this is similar to https://github.com/papercups-io/papercups/issues/95, correct?

would you like to completely hide the chat widget when it's "off", or just display a message indicating you aren't available?

**Comment by jamesefhawkins** [comment]
I'd probably rather hide it completely so we don't disappoint people, but either way seems kind of valid.

**Comment by reichert621** [comment]
got it, we can support both and let you choose whichever you prefer 👍 hoping to get to this by next week!

**Comment by reichert621** [comment]
@jamesefhawkins out of curiosity, how "complicated" are your working hours? 

e.g. is it something pretty straightforward like "weekdays from 9am-5pm", or is it more like "mondays from 10-6, tuesdays from 9-7, etc etc"?

(just trying to get a sense of how we should design this UX!)
